pkg/bgp/speaker/speaker.go

```diff
@@ -128,7 +128,7 @@ func (s *MetalLBSpeaker) OnUpdateService(svc *slim_corev1.Service) error {
 	}
 
 	l.Debug("adding event to queue")
-	s.queue.Add(epEvent{
+	s.queue.Add(svcEvent{
```

Comment:
good citizen change, this is mislabeled but causes no functional difference in the code, other then logging the wrong event type.